packages/attribute-registry-consumer/src/attributeRegistryConsumerService.ts

```diff
@@ -0,0 +1,34 @@
+import { match } from "ts-pattern";
+import {
+  logger,
+  consumerConfig,
+  ReadModelRepository,
+} from "pagopa-interop-commons";
+import { EventEnvelope } from "./model/models.js";
+import { fromAttributeV1 } from "./model/converter.js";
+
+const { attributes } = ReadModelRepository.init(consumerConfig());
+
+export async function handleMessage(message: EventEnvelope): Promise<void> {
+  logger.info(message);
+  await match(message)
+    .with({ type: "AttributeAdded" }, async (msg) => {
+      await attributes.updateOne(
```

Comment:
I think a performance increase like that isn't that significant for this use case.
Upsert could even be dangerous because it means bypassing a constraint: if I add a new document I have to be sure that it doesn't already exist, otherwise it means there is a problem of which we would have no trace, for example, duplication of events.

Reply:
Ok, I understand the reason for that choice.
Let's keep `updateOne`.